index.html

```diff
@@ -4,34 +4,65 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <title>Project 1</title>
+    <style>
+        h1,
+        h3 {
+            font-family: Arial, Helvetica, sans-serif;
+            color: #f4e1d2;
+        }
+
+        h1 {
+            text-decoration: underline;
+        }
+
+        h3 {
+            font-style: italic;
+        }
+
+        p.hi {
+            font-size: 15px;
+        }
+    </style>
 </head>
 
 <body>
-    on paper
-    by Jacqueline Woodson
+    <div class="container" id="container">
+        <div id="container3">
+            <h1 id="text2">On Paper</h1>
+            <h3 id="text3">by Jacqueline Woodson</h3>
```

Comment:
Here, you have skipped a heading level. Headings provide organization to your HTML document and [you should not skip levels](https://mica-web.github.io/learn/html/best-practices#heading-tags-are-used-semantically-not-simply-for-presentation), which means line 34 should be an `h2`.